admin_pages/messages/templates/ee_msg_messages_shortcodes_help_tab.template.php

```diff
@@ -17,5 +21,5 @@
 <?php
 foreach ($shortcodes as $name => $description) : ?>
     <p><strong><?php echo esc_html($name); ?></strong></p>
-    <p><?php echo $description; // already escaped ?></p>
+    <p><?php echo wp_kses($description, AllowedTags::getWithFormTags()); ?></p>
 <?php endforeach; ?>
```

Comment:
can you plz add a new line at the end of this file?

surprised the lint check didn't catch this